deno.json

```diff
@@ -30,9 +29,8 @@
     "tailwindcss": "npm:tailwindcss@3.4.1",
     "tailwindcss/": "npm:/tailwindcss@3.4.1/",
     "tailwindcss/plugin": "npm:/tailwindcss@3.4.1/plugin.js",
-    "std/": "https://deno.land/std@0.208.0/",
+    "$std/": "https://deno.land/std@0.208.0/",
```

Comment:
This is the std import that comes with a Fresh install anyway.

Reply:
Very true.